src/components/Header/Header.js

```diff
@@ -2,32 +2,92 @@ import React, { PureComponent } from 'react'
 import PropTypes from 'prop-types'
 
 import Description from '../Description/Description'
-import DownloadButton from '../DownloadButton/DownloadButton'
+import ExternalLink from '../ExternalLink/ExternalLink'
 import './Header.scss'
 
 export default class Header extends PureComponent {
   render () {
-    const { title, version, description, specUrl } = this.props
+    const { title, version, description, info, specUrl } = this.props
 
     return (
       <header id='header'>
         <h1>{title}</h1>
-        <Description description={description} />
-        <div>
-          <DownloadButton url={specUrl}>Download this Open API definition.</DownloadButton>
-        </div>
+
         <dl className='inline-pairs'>
           <dt>Version</dt>
           <dd>{version}</dd>
+          {info && info.contact && this.renderContact(info.contact)}
+          {info && info.license && this.renderLicense(info.license)}
         </dl>
+
+        {description && <Description description={description} />}
+
+        <nav>
+          <ExternalLink href={specUrl}>Download OpenAPI definition</ExternalLink>
+          {info && info.termsOfService &&
+          <ExternalLink href={info.termsOfService}>Terms of Service</ExternalLink>}
+        </nav>
       </header>
     )
   }
+
+  /**
+   * The contact object has no required fields, so the logic is a bit meh as
+   * they are several permutations to consider.
+   *
+   * @param {Object} contact
+   */
+  renderContact (contact) {
+    const elements = [<dt key='contact'>Contact</dt>]
+
+    if (contact.name && contact.url) {
+      elements.push(
+        <dd key='contact-value'>
+          <ExternalLink href={contact.url}>{contact.name}</ExternalLink>
+          {contact.email && `(${contact.email})`}
+        </dd>
+      )
+    } else if (contact.name) {
+      elements.push(
+        <dd key='contact-value'>{contact.name}</dd>
+      )
+    } else if (contact.url) {
+      elements.push(
+        <dd key='contact-value'>
+          <ExternalLink href={contact.url}>{contact.url}</ExternalLink>
+          {contact.email && `(${contact.email})`}
+        </dd>
+      )
+    } else if (contact.email) {
+      elements.push(
+        <dd key='contact-value'>
+          <ExternalLink href={`mailto:${contact.email}`}>{contact.email}</ExternalLink>
+        </dd>
+      )
+    }
+
+    return elements
```

Comment:
Mmm, I'm not sure there's much to gain. I don't know if the example would actually work either?